tests/testthat/_snaps/model-options.md

```diff
@@ -3,30 +3,30 @@
     Code
       print_choices(meas_choices(), format = TRUE)
     Output
-      [1] "\"lcdm\", \"dina\", \"dino\", \"crum\", \"nida\", \"nido\", or \"ncrum\""
+      [1] "\"lcdm\", \"dina\", \"dino\", \"crum\", \"nida\", or \"nido\""
     Code
       print_choices(strc_choices(), format = TRUE)
     Output
       [1] "\"unconstrained\" and \"independent\""
     Code
       print_choices(meas_choices(), last = ", and ", format = TRUE)
     Output
-      [1] "\"lcdm\", \"dina\", \"dino\", \"crum\", \"nida\", \"nido\", and \"ncrum\""
+      [1] "\"lcdm\", \"dina\", \"dino\", \"crum\", \"nida\", and \"nido\""
 
 ---
 
     Code
       print_choices(meas_choices())
     Output
-      [1] "lcdm, dina, dino, crum, nida, nido, or ncrum"
+      [1] "lcdm, dina, dino, crum, nida, or nido"
```

Comment:
`ncrum` should still be an option, just as `crum` is an acceptable option. It just wraps `meas_lcdm()` on the back end.